### PR TITLE
Add support for the Clog

### PR DESCRIPTION
**Pull request by smores56**
## Board/Shield Check-list
 - [x] This board/shield is tested working on real hardware
 - [x] Definitions follow the general style of other shields/boards upstream ([Reference](https://zmk.dev/docs/development/new-shield))
 - [x] `.zmk.yml` metadata file added
 - [x] Proper Copyright + License headers added to applicable files (Generally, we stick to "The ZMK Contributors" for copyrights to help avoid churn when files get edited)
 - [x] General consistent formatting of DeviceTree files
 - [x] Keymaps do not use deprecated key defines (Check using the [upgrader tool](https://zmk.dev/docs/codes/keymap-upgrader))
 - [x] `&pro_micro` used in favor of `&pro_micro_d/a` if applicable
 - [x] If split, no name added for the right/peripheral half
 - [x] Kconfig.defconfig file correctly wraps *all* configuration in conditional on the shield symbol
 - [x] `.conf` file has optional extra features commented out
 - [x] Keyboard/PCB is part of a shipped group buy or is generally available in stock to purchase (OSH/personal projects without general availability should create a zmk-config repo instead)

These boards (both the Clog and its MX counterpart, the Sephirette) are available on my store, smoresboards.com. Both have been built and tested with ZMK using the current files.

### CODE REVIEW

File: app/boards/shields/clog/clog.conf [hunk 1]
```diff
@@ -0,0 +1 @@
+CONFIG_ZMK_KSCAN_DEBOUNCE_RELEASE_MS=10
```
[the remark this answers is not in the document]
Comment (Nicell) [reply app/boards/shields/clog/clog.conf:1]: Looking a little further, this shield uses direct wiring. We need to add similar debouncing we have on matrices to direct wires!

File: app/boards/shields/clog/clog_left.conf [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# Copyright (c) 2020 The ZMK Contributors
```
Comment (Nicell) [inline app/boards/shields/clog/clog_left.conf:1]: Please remove this `.conf` as well as the `_right.conf`. Only the base `.conf` should be used.

**Review by Nicell — CHANGES_REQUESTED** [review]
Got a few comments, sweet looking keyboards!

File: app/boards/shields/clog/clog.conf (hunk 1 above)
[the remark this answers is not in the document]
Comment (smores56) [reply app/boards/shields/clog/clog.conf]: This is inconsistently necessary. With it enabled, there is definitely no problem.

**Review by Nicell — APPROVED** [review]
Thanks for the contribution!

File: app/boards/shields/clog/clog.conf (hunk 1 above)
[the remark this answers is not in the document]
Comment (caksoylar) [reply app/boards/shields/clog/clog.conf]: Like @Nicell implied, I don't think this setting does anything for direct pin keyboards. The debounce overhaul only applied to matrix scan, for direct pins the old way of setting debounce period works:
```devicetree
&kscan0 {
    debounce-period = <10>;
};
```

[the remark this answers is not in the document]
Comment (Nicell) [reply app/boards/shields/clog/clog.conf]: Yep, @caksoylar is right. I've started an issue to track the hopeful implementation of the debounce improvements being added to direct wire with #1104.

**Pull request merged**